contracts/SnappAuction.sol

```diff
@@ -58,8 +58,8 @@ contract SnappAuction is SnappBase {
         uint128 sellAmount
     ) public onlyRegistered() {
         // Must have 0 < tokenId < MAX_TOKENS anyway, so may as well ensure registered.
-        require(tokenIdToAddressMap[buyToken] != address(0), "Buy token is not registered");
-        require(tokenIdToAddressMap[sellToken] != address(0), "Sell token is not registered");
+        require(buyToken < numTokens, "Buy token is not registered");
```

Comment:
Why that change?

Reply:
Because I was not able to access     
```
BiMap.Data private registeredAccounts;
BiMap.Data private registeredTokens;
```
from within the Auction contract, and these are equivalent statements anyway.